ansible/roles/baremetal/tasks/pre-install.yml

```diff
@@ -149,7 +148,6 @@
   file:
     path: "{{ node_config_directory }}"
     state: directory
-    recurse: yes
-    mode: 0644
+    mode: 0755
```

Comment:
why do others need access to this directory?

Reply:
This is /etc/kolla/, which in an AIO environment is also where the localhost config goes, so may need to be readable by a different user. The change here is to add the executable bit.